lib/src/util/__tests__/keyboardUtil.test.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useState } from 'react';
+import { rtlRender } from '../../__testutils__/rtlUtils';
+import { userEvent } from '../../__testutils__/rtlUtils';
+import { modifierKeyIsPressed } from '../keyboardUtil';
+
+describe('modifierKeyIsPressed', function () {
+  const Fixture = () => {
+    const [isPressed, setIsPressed] = useState(false);
+    const handleKeyDown = (event: React.KeyboardEvent) => {
+      event.preventDefault();
```

Comment:
Why do we need to `preventDefault` here?

Reply:
> Oh that's so userEvent.keyboard doesn't lose focus on the fixture.

What's causing it to lose focus without this?  If your test was pressing Tab I'd assume it's that, but that's not what you're doing.

Reply:
> > Oh that's so userEvent.keyboard doesn't lose focus on the fixture.
> 
> What's causing it to lose focus without this? If your test was pressing Tab I'd assume it's that, but that's not what you're doing.

I'm not entirely sure but `userEvent.keyboard` gives me this error:
`Cannot read properties of null (reading 'nodeType')` (Sounded like it lost focus)
The error occur when attempting {Modifier+ArrowLeft} after {Modifier+ArrowUp} is pressed.

Reply:
> I'm not entirely sure but userEvent.keyboard gives me this error:
Cannot read properties of null (reading 'nodeType') (Sounded like it lost focus)
The error occur when attempting {Modifier+ArrowLeft} after {Modifier+ArrowUp} is pressed.

Upgrading to the latest `@testing-library/user-event` seems to fix that